node/src/main/java/de/obsidiancloud/node/command/ScreenCommand.java

```diff
@@ -25,20 +27,20 @@ public void execute(@NotNull CommandExecutor executor, @NotNull String[] args) {
         OCServer server = ObsidianCloudAPI.get().getServer(args[0]);
         if (server == null) {
             executor.sendMessage("§cThe server §e" + args[0] + " §cdoes not exist.");
-            return;
-        }
-
-        if (server instanceof LocalOCServer localServer) {
-            if (localServer.isScreen()) {
-                localServer.setScreen(false);
+        } else if (server instanceof LocalOCServer localServer) {
```

Comment:
Speicher screenreaders in ner variable ab und wiederverwende es

`Set<CommandExecutor> screenReaders = localServer.getScreenReaders()`